crates/completest-pty/src/lib.rs

```diff
@@ -213,6 +213,8 @@ PS1='% '
         let mut command = Command::new("bash");
         command
             .env("PATH", &self.path)
+            // Don't load `~/.inputrc` which could set the vi edit mode.
+            .env("INPUTRC", "/etc/inputrc")
```

Comment:
Is there a way to disable this completely, rather than redirect to a specific one?  How universal is that location (e.g. Mac)?

Reply:
Per [manpages](https://www.man7.org/linux/man-pages/man1/bash.1.html#READLINE), if `INPUTRC` is not found, it will load the default value (on linux it is `/etc/inputrc`).
There is `--noediting`, but it may disable bash completion. There's also `set +o vi` to be set in `.bashrc`,
but `~/.inputrc` still override it somehow.

Reply:
What if `BashRuntime::new` instead put a `.inputrc` in our "home" that is empty and in this code we the environment variable to point to that?

Reply:
Oh, that seems to work.